uvicorn/protocols/http.py

```diff
@@ -247,10 +249,11 @@ def on_body(self, body: bytes):
         self.body = body
 
     def on_message_complete(self):
-        self.body_queue({
-            'type': 'http.request',
-            'body': self.body
-        })
+        if self.active_request:
```

Comment:
Note that ‘body_queue’ and ‘active_request’ don’t (necessarily) point to the same thing. (The request currently being parsed by the server vs. the request currently being handled by a client)

I guess if we value clarity rather than microoptimizations then I could make that clearer in the codebase.